eiger_io/fs_handler.py

```diff
@@ -20,3 +21,53 @@ def __call__(self, seq_id):
         master_path = '{}_{}_master.h5'.format(self._base_path, seq_id)
         # TODO Return a multi-dimensional PIMS seq.
         return np.array(EigerImages(master_path))
+
+
+class FixedEigerImages(EigerImages):
```

Comment:
Yes, that's the plan. @sameera2004 I pasted this into the notebook for expedience. What we should do now is add these properties to the `EigerImages` class in `eiger_io/pims_reader.py`.